src/prng/mod.rs

```diff
@@ -59,82 +59,72 @@
-//! | [`Isaac64Rng`] | fast | 4136 bytes | slow          | unknown        | unknown        |
+//! | name | full name |  performance | initialization | memory | predictability | backtracking resistance |
+//! |------|-----------|--------------|--------------|----------|----------------|-------------------------|
+//! | [`ChaChaRng`] | ChaCha20 | ⭐⭐⭐ | ⭐⭐⭐⭐⭐ | 136 bytes | secure | no |
+//! | [`Hc128Rng`] | HC-128 | ⭐⭐⭐⭐⭐ | ⭐ | 4176 bytes | secure | no |
+//! | [`IsaacRng`] | ISAAC | ⭐⭐⭐⭐ | ⭐ | 2072 bytes | unknown | unknown |
+//! | [`Isaac64Rng`] | ISAAC-64 | ⭐⭐⭐⭐⭐ | ⭐ | 4136 bytes| unknown | unknown |
 //!
 //! It should be noted that the ISAAC generators are only included for
-//! historical reasons, originally being selected as a "compromise" between
-//! security and performance. They have good quality output and no attacks are
-//! known, but have received little review from cryptography experts.
+//! historical reasons, they have been with the Rust language since the very
+//! beginning. They have good quality output and no attacks are known, but have
+//! received little attention from cryptography experts.
 //!
 //!
 //! # Performance
 //!
-//! First it has to be said most PRNGs are very fast, and will
-//! rarely be a performance bottleneck. Performance of PRNGs is however a bit of
-//! a subtle thing. It depends a lot on the CPU architecture (32 vs. 64 bits),
-//! inlining, and also on the number available of registers. The performance of
-//! simple PRNGs is often affected by surrounding code due to inlining and
-//! other usage of registers.
+//! First it has to be said most PRNGs are very fast, and will rarely be a
+//! performance bottleneck.
 //!
-//! When choosing a PRNG for performance it is very important to benchmark your
-//! own application due to interactions between PRNGs and surrounding code and
+//! Performance of normal PRNGs is a bit of a subtle thing. It depends a lot on
+//! the CPU architecture (32 vs. 64 bits), inlining, and also on the number of
+//! available registers. This often causes the performance to be affected by
+//! surrounding code due to inlining and other usage of registers.
+//!
+//! When choosing a PRNG for performance it is important to benchmark your own
+//! application due to interactions between PRNGs and surrounding code and
 //! dependence on the CPU architecture as well as the impact of the size of
 //! data requested. Because of all this, we do not include performance numbers
-//! here but merely a qualitative description.
+//! here but merely a qualitative rating (which is not comparable between PRNGs
+//! and CSPRNGs).
 //!
 //! CSPRNGs are a little different in that they typically generate a block of
 //! output in a cache, and pull outputs from the cache. This allows them to have
 //! good amortised performance, and reduces or completely removes the influence
-//! of surrounding code on the CSPRNG performance.
-//!
-//! ## Worst-case performance
```

Comment:
Please keep this section. Some people do care about worst-case performance and don't want to have to read every line to find the little note.